transport/server.cc

```diff
@@ -224,6 +224,9 @@ cql_server::listen(socket_address addr, std::shared_ptr<seastar::tls::credential
     }
     return f.then([this, addr, keepalive](shared_ptr<seastar::tls::server_credentials> creds) {
         listen_options lo;
+        if (_config.cpu_sharding_algorithm_name == "biased-token-source-port"){
```

Comment:
Is it necessary to compare strings on every new connection? Is there any reason not to create a bool variable computed once and check it instead?

Reply:
Or maybe use enum instead of streams?

Reply:
I'll look into how convert it internally to an enum

Reply:
Even better just have a variable in cql_server of type load_palancing_algorithm, set it once, and then just use it to initialize lo.lba.